src/main/java/org/zendesk/client/v2/model/hc/ArticleAttachments.java

```diff
@@ -74,7 +77,6 @@ public void setUrl(String url) {
         this.url = url;
     }
 
-    @JsonProperty("article_id")
```

Comment:
Why did you remove this annotation for random properties, but not for all? I don't suggest deleting it everywhere. We should probably return it to ensure that it is consistent with the rest of the content in the file.